src/components/Form/FormFields/FormField.tsx

```diff
@@ -54,6 +55,16 @@ const FormField = ({
   field?: FormFieldBaseProps<any>;
   children: React.ReactNode;
 }) => {
+  const [phhelp, setPhhelp] = useState(false);
```

Comment:
using this effect I am just checking which compnent I am using if it is coming from searchBymultified then help is false  and no need to show it ,excepting only this serchbymultifield other serch field will show this help section as previous
![Screenshot 2024-10-31 034754](https://github.com/user-attachments/assets/f746f1a8-2874-48fe-91c8-e132c2ae66fb)
![Screenshot 2024-10-31 034828](https://github.com/user-attachments/assets/3c00cd1a-2435-4225-9774-009c0c1d4844)